src/app/modules.ts

```diff
@@ -25,13 +25,15 @@ import {
     ToolbarModule,
     TreeTableModule
 } from 'primeng/primeng';
+import { DotTextareaContentModule } from './view/components/_common/dot-textarea-content/dot-textarea-content.module';
```

Comment:
Just for the pattern library, after @hmoreras changes we can move it to the pattern library module.